.github/workflows/tests.yml

```diff
@@ -4,11 +4,7 @@
 name: Tests
 
 on:
-  push:
```

Comment:
Feel like we should revert this part. Tests are running 2ce without the limit. I think you use it to run on the fork, right? Maybe there is a better way to configure Actions...

Reply:
Reverted in #669